components/GoalsCard/GoalsCard.js

```diff
@@ -0,0 +1,20 @@
+import { StyledItem, StyledList, StyledGoalCard, StyledText } from "@/styles";
+
+export default function GoalsCard({ goals }) {
+  return (
+    <>
+      <StyledList>
+        {goals.map(({ id, goalName, savedAmount, goalAmount }) => (
+          <StyledItem key={id}>
+            <div>
+              <StyledGoalCard>{goalName}</StyledGoalCard> <br></br>{" "}
+              <StyledGoalCard>
+                saved {savedAmount} EUR of {goalAmount} EUR
+              </StyledGoalCard>
+            </div>
+          </StyledItem>
+        ))}
+      </StyledList>
+    </>

```

Comment:
We don't need the React Fragment if we are only returning one thing from this component (i.e. `StyledList`).

Also...

While using `<br></br>` is quick and straightforward for spacing, it's generally better to control spacing through CSS for a more flexible and responsive design.

```suggestion

      <StyledList>
        {goals.map(({ id, goalName, savedAmount, goalAmount }) => (
          <StyledItem key={id}>
            <div>
              <StyledGoalCard>{goalName}</StyledGoalCard> <br></br>{" "}
              <StyledGoalCard>
                saved {savedAmount} EUR of {goalAmount} EUR
              </StyledGoalCard>
            </div>
          </StyledItem>
        ))}
      </StyledList>

```